modules/calib3d/src/calibration.cpp

```diff
@@ -1299,18 +1299,15 @@ CV_IMPL double cvCalibrateCamera2( const CvMat* objectPoints,
         (npoints->rows != 1 && npoints->cols != 1) )
         CV_Error( CV_StsUnsupportedFormat,
             "the array of point counters must be 1-dimensional integer vector" );
-    if(flags & CV_CALIB_TILTED_MODEL)
-    {
+    if(flags & CV_CALIB_TILTED_MODEL){
```

Comment:
Please, revert braces formatting to make it consistent with the surrounding code.